weave-js/src/components/PagePanelComponents/Home/Browse3.tsx

```diff
@@ -392,37 +396,33 @@ const MainPeekingLayout: FC = () => {
           <WeaveflowPeekContext.Provider value={{isPeeking: true}}>
             <SimplePageLayoutContext.Provider
               value={{
-                headerPrefix: (
-                  <>
-                    <Box
-                      sx={{
-                        flex: '0 0 auto',
-                        height: '47px',
-                      }}>
-                      <IconButton
-                        onClick={() => {
-                          closePeek();
-                        }}>
-                        <Close />
-                      </IconButton>
-                    </Box>
-                    <Box
-                      sx={{
-                        flex: '0 0 auto',
-                        height: '47px',
-                      }}>
-                      <IconButton
-                        onClick={() => {
-                          const targetPath = query.peekPath!.replace(
-                            generalBase,
-                            targetBase
-                          );
-                          history.push(targetPath);
-                        }}>
-                        <Fullscreen />
-                      </IconButton>
-                    </Box>
-                  </>
+                headerSuffix: (
+                  <Box
+                    sx={{
+                      height: '47px',
+                      flex: '0 0 auto',
+                    }}>
+                    <Button
+                      tooltip="Open full page for this object"
+                      icon="full-screen-mode-expand"
+                      variant="ghost"
+                      className="ml-4"
+                      onClick={() => {
+                        const targetPath = query.peekPath!.replace(
```

Comment:
I am not sure exactly why by it seems that this expansion drops the trace show/hide state

Reply:
It's because I'm keeping the show/hide state in a query param and we're not passing search to history.push.

I noticed this and thought maybe it made sense for it to fall back to showing the trace tree when you expand because there is so much more space, but maybe it's too confusing. I'll change it to preserve the state.